dolly-backend-app/src/main/java/no/nav/dolly/bestilling/service/GjenopprettGruppeService.java

```diff
@@ -63,7 +83,34 @@ public void executeAsync(Bestilling bestilling) {
                                 List<Person> personer = tpsfService.hentTestpersoner(List.of(testident.getIdent()));
 
                                 if (!personer.isEmpty()) {
-                                    sendTestidenter(bestilling, bestKriterier, progress, personer.get(0));
+                                    TpsPerson tpsPerson = tpsfPersonCache.prepareTpsPersoner(personer.get(0));
+                                    sendIdenterTilTPS(new ArrayList<>(List.of(bestilling.getMiljoer().split(","))),
```

Comment:
Hvorfor lager du en liste med `List.of` så gjør du om listen til en ny arrayListe? 
`new ArrayList<>(List.of(bestilling.getMiljoer().split(",")))`

Hvis du bare ønsker at listen mutable kan du bruke: `Arrays.asList(bestilling.getMiljoer().split(","))`

Reply:
👍